services/core-api/app/api/verifiable_credentials/manager.py

```diff
@@ -195,3 +288,116 @@ def produce_map_01_credential_payload(cls, did: str, permit_amendment: PermitAme
+        for pa in permit_amendment.permittee_appointments:
+            if curr_appt.start_date < pa.start_date:
+                curr_appt = pa
+            else:
+                break
+        orgbook_entity = curr_appt.party.party_orgbook_entity
+        if not orgbook_entity:
+            current_app.logger.warning("No Orgbook Entity, do not produce Mines Act Permit UNTP CC")
+            return None
+
+        untp_party_cpo = base.Party(
+            name="Chief Permitting Officer",
+            identifiers=[
+                base.Identifier(
+                    scheme=ANONCRED_SCHEME,
+                    identifierValue="did:indy:candy:A2UZSmrL9N5FDZGPu68wy",
+                    identifierURI="https://candyscan.idlab.org/tx/CANDY_PROD/domain/321",
+                    verificationEvidence=base.Evidence(
+                        format=codes.EvidenceFormat.W3C_VC,
+                        credentialReference=
+                        "https://candyscan.idlab.org/tx/CANDY_PROD/domain/321"            #this is an anoncred
+                    ))
+            ])
+        orgbook_cred_url = f"https://orgbook.gov.bc.ca/entity/{orgbook_entity.registration_id}/credential/{orgbook_entity.credential_id}"
+
+        untp_party_business = base.Party(
+            name=orgbook_entity.name_text,
+            identifiers=[
+                base.Identifier(
+                    scheme=ANONCRED_SCHEME,
+                    identifierValue=str(orgbook_entity.registration_id),
+                    identifierURI=orgbook_cred_url,
+                    verificationEvidence=base.Evidence(
+                        format=codes.EvidenceFormat.W3C_VC, credentialReference=orgbook_cred_url))
+            ])
+
+        facility = cc.Facility(
+            name=permit_amendment.mine.mine_name,
+            geolocation=
+            f'https://plus.codes/{plus_code_encode(permit_amendment.mine.latitude, permit_amendment.mine.longitude)}',
+            verifiedByCAB=True)
+
+        products = [
+            cc.Product(
+                name=c,
+                classifications=cc.Classification(
+                    scheme="https://unstats.un.org/unsd/classifications/Econ/cpc",
+                    classifierName=c),
+                verifiedByCAB=False) for c in permit_amendment.mine.commodities
+        ]
+
+        untp_assessments = [
+            cc.ConformityAssessment(
+                referenceRegulation=cc.Regulation(
+                    id="https://www.bclaws.gov.bc.ca/civix/document/id/complete/statreg/96293_01",
+                    name="BC Mines Act",
+                    issuingBody=base.Party(
+                        name="BC Government",
+                        identifiers=[
+                            base.Identifier(identifierValue="did:indy:candy:LTNyw5R14J66CrF7tmV3i8")
+                        ]),
+                    effectiveDate=datetime(2024, 5, 14, tzinfo=pytz.timezone("UTC")).isoformat()),
```

Comment:
Is the effectiveDate supposed to be hardcoded?

Reply:
yes. that date is date the Mine Act was Passed. however now the date shown in the webpage has changed again..... so maybe we need a different date. 

https://www.bclaws.gov.bc.ca/civix/document/id/complete/statreg/96293_01

Reply:
That makes sense!